services/cms/src/utils/Gutenberg/modifyBlockAttributes.ts

```diff
@@ -1,6 +1,6 @@
 import { assign } from "lodash"
 
-export function modifyBlockAttributes(settings, name) {
+export function modifyBlockAttributes(settings: any, name: any) {
```

Comment:
```suggestion
export function modifyBlockAttributes(settings: any, name: string) {
```
also add ignore for the any